edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -0,0 +1,1349 @@
+
+Closing/extending the registration period:
+
+- If you need to close registration earlier than expected, you may do so via the button on the top of the "organizer view" section of the competition website. If you want to close registration before the competitor limit has been filled, you may do so by emailing WCAT with a request and explanation.
+- Extending the registration period: write an email on the competition thread and ask for an extension. The email must include the closing time in UTC and the date. You must also write the reasons for the extension. A valid reason could be that you have fewer registrations than expected.
+- Additional registration periods must be announced at least 24 hours before they are scheduled to start. The request must be written to WCAT with times and dates in UTC. A valid reason to re-open registration could be that you have fewer registrations than expected. This approach can also be used for proposing on-the-spot registration (OTS).
+
+Increasing the competitor limit:
+
+- Delegates can request a change to the competitor limit of a competition by providing a valid reason. Potential reasons for increasing the competitor limit are that you have a larger venue than initially expected or the competitor to guest ratio is lower than expected or that you chose a competitor limit that was too low to begin with. Do not increase the competitor limit if there’s any likelihood of it being crammed in the venue. Registrations must **never** be accepted before WCAT approves the limit increase.
+
+## Managing Registrations
+
+Managing registrations is a responsibility of the organizer. However, as a Delegate, you must ensure that the organizers of the competition are following through with this responsibility. Regularly confirm that:
+
+- New registrations are being approved.
+- Only competitors who have paid for registration are being accepted, excluding those exempt from registration fees according to [point 1.3 of the Code of Ethics](https://documents.worldcubeassociation.org/documents/Code%20of%20Ethics.pdf).
+- Organizers are accepting registrations in the order of registration, and not showing favoritism or exclusionary practices based on any factor.
+- Refunds are being handled, especially if your competition allows registrants to delete their own registration.
+- Newcomers have their names capitalized and formatted correctly.
+- All registration requirements and deadlines you have set are being followed.
+- No registered competitors are on the [Banned Competitors list](https://www.worldcubeassociation.org/disciplinary). If a competitor who is banned registers for a competition, contact WDC immediately.
```

Comment:
Is this the right link? I thought it would be a link to an actual list of banned competitors.

Reply:
This was updated within the last couple weeks. I've just updated the link in 048300a.